src/postgraphql/postgraphql.ts

```diff
@@ -130,6 +131,7 @@ export default function postgraphql (
     try {
       const pgClient = await pgPool.connect()
       const newGqlSchema = await createPostGraphQLSchema(pgClient, pgSchemas, options)
+      await exportPostGraphQLSchema(newGqlSchema, options)
```

Comment:
I am a little concerned that we have to wait for the files to be written before resolving `createGqlSchema`. Since it looks like we already catch and log errors here, could we remove `await` and add a `.catch(() => { ... })` which logs the error and exits the process? (like below)